packages/components/src/internal/components/container/FolderAPIWrapper.ts

```diff
@@ -17,11 +17,16 @@ export interface ProjectSettingsOptions {
     title?: string;
 }
 
+export interface ProjectLAFOptions {
```

Comment:
nit: Consider renaming to `UpdateProjectSettingsOptions` so it is consistent with the endpoint.